python/ray/serve/tests/test_fastapi.py

```diff
@@ -53,7 +53,7 @@ class FastAPIApp:
 
     resp = requests.get("http://localhost:8000/f/not-number")
     assert resp.status_code == 422  # Unprocessable Entity
-    assert resp.json()["detail"][0]["type"] == "type_error.integer"
+    assert resp.json()["detail"][0]["type"] in {"type_error.integer", "int_parsing"}
```

Comment:
Add a comment explaining that `type_error.integer` is used in Pydantic 1.x, and `int_parsing` is used in Pydantic 2.x.